mcr_object_recognition_mean_circle/README.md

```diff
@@ -1 +1,32 @@
 The current model is trained with sklearn version 0.14.1.
+
+### How to use
+## Install sklearn
+
+Run:
+pip install sklearn
```

Comment:
Are this really required, our dependencies should be managed with `rosdep`, not manually with pip

Reply:
@sthoduka would it be better here to have a `requirements.txt` that you can refer to in the `README`?

Reply:
yep